### DISCUSSION TITLE
🙏 how to recover public keys for webauthn with the api ?

### DISCUSSION THREAD

**Discussion by Romain-Fromantin** [ory/kratos#2700] [Q&A]
I can’t find webauthn in ui.nodes
how the user interface(of example)  does it to have it?

**Comment by vinckr** [comment]
Hello @romain63110 

can you tell me a bit more about what you are trying to do, what is the exact use case that you need the public keys for? 
Could be that WebAuthn is not a part of the ui.nodes, I am not sure. 

Are you running Ory Kratos yourself or using the managed service? What frontend are you using?
